_sources/Classes/sorting_instances.rst

```diff
@@ -64,3 +67,58 @@ Sometimes you will find it convenient to define a method for the class that does
    for f in sorted(L, key=lambda x: x.sort_priority()):
        print(f.name)
 
+
+Approach 2: Defining Sort Orders with Comparison Operators
+----------------------------------------------------------
+
+Another approach to sorting lists of instances is to specify a "comparison operator" for the class---a method that takes two instances as arguments and "decides" which should come first. One advantage of this approach is that you can call ``sorted`` on a list of instances **without** specifying a value for ``key`` and it will sort in the order you defined.
+
+To do this, we can define a method named ``__lt__`` which stands for "less than". Note that this method starts and ends with two underscores. This signifies that it is a special method, just like ``__init__`` and ``__str__``. Our method, ``__lt__``, takes two instances as arguments: ``self`` and an argument for another instance. It returns ``True`` if the ``self`` instance should come before the other instance, and ``False`` otherwise. Normally, ``__lt__`` is called when we try to use the less than operator (``<``) on class instances; Python translates the expression ``a < b`` into ``a.__lt__(b)``. However, we can also use ``__lt__`` to decide which of two instances should come first in a sorted list. For example, if we wanted to sort instances of ``Fruit`` by prices names by default, we could define ``__lt__`` as follows:
```

Comment:
In the last sentence, you have "prices names". Presumably, you just want to delete "names" there.

Reply:
Yes, will submit a new PR